src/io/flutter/perf/WidgetPerfLinter.java

```diff
@@ -63,7 +63,7 @@
       "perf_diagnosis_demo/lib/list_demo.dart",
       "Using ListView to load items efficiently",
       "listViewLoad",
-      "https://master-docs-flutter-io.firebaseapp.com/flutter/widgets/ListView-class.html#child-elements-lifecycle",
+      "https://docs.flutter.io/flutter/widgets/ListView-class.html#child-elements-lifecycle",
```

Comment:
For the ListView, it might be more useful to just send the user to the beginning of the page: https://docs.flutter.io/flutter/widgets/ListView-class.html

The key information is the first bullet point on that page: "The default constructor takes an explicit List<Widget> of children. This constructor is appropriate for list views with a small number of children because constructing the List requires doing work for every child that could possibly be displayed in the list view instead of just those children that are actually visible."

Reply:
done